includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -350,6 +350,25 @@ public static function get_testing_instructions_for_optimized_checkout() {
+	 * @param string $instructions The testing instructions containing <number> tags.
+	 * @return string Instructions with <number> tags replaced by copy button HTML.
+	 */
+	public static function expand_copy_button_markup( $instructions ) {
+		$copy_label = esc_attr__( 'Copy to clipboard', 'woocommerce-gateway-stripe' );
+
+		return preg_replace(
+			'/<number>(.*?)<\/number>/',
+			'<button type="button" class="js-wc-stripe-copy-test-number" aria-label="' . $copy_label . '" title="' . $copy_label . '"><i></i><span>$1</span></button>',
+			$instructions
+		);
+	}
```

PHPMD (2.15.0)

[error] The method expand_copy_button_markup is not named in camelCase. (undefined)

(CamelCaseMethodName)